.github/workflows/dotnet.yml

```diff
@@ -0,0 +1,36 @@
+name: Chorus .NET6
+
+on:
+  [push, pull_request, workflow_dispatch]
+
+jobs:
+  build_test_deploy:
+    env:
+      NUGET_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
+    runs-on: windows-latest
```

Comment:
We should also build on Linux

Reply:
I wasn't able to get the Linux side to work. the file system was missing mercurial lib's, if i remember

Reply:
Would installing the SIL.Chorus.Mercurial nuget package help, or `sudo apt install mercurial` ?

Reply:
I see SIL.Chorus.Mercurial is already installed. In my local testing I had to install `sudo apt install python2` to get things working.

Reply:
Depending on the version of SIL.Chorus.Mercurial and/or the version of Ubuntu, you might also need to run `sudo apt install python-is-python2` (that was necessary in the Microsoft .NET SDK images, for example). That one installs `python2` *and* creates a `/usr/bin/python` symlink to `python2`. (But in newer versions of SIL.Chorus.Mercurial, the Mercurial script is updated to look for `/usr/bin/python2` instead of `/usr/bin/python` in which case you can just install the `python2` package and be fine).